lib/ansible/modules/database/postgresql/postgresql_privs.py

```diff
@@ -617,12 +653,19 @@ def manipulate_privs(self, obj_type, privs, objs, roles,
 
             for_whom = ','.join(for_whom)
 
+        # as_who:
+        if target_roles:
+            as_who = ','.join(pg_quote_identifier(r, 'role') for r in target_roles)
+        else:
+            as_who = target_roles
```

Comment:
```suggestion
          as_who = None
          if target_roles:
            as_who = ','.join(pg_quote_identifier(r, 'role') for r in target_roles)
```

This `as_who = target_roles` is confusing, since `target_roles` is None, empty string or something falsy in general.